packages/core/src/modules/credentials/messages/IssueCredentialMessage.ts

```diff
@@ -48,13 +47,8 @@ export class IssueCredentialMessage extends AgentMessage {
   public get indyCredential(): Cred | null {
     const attachment = this.credentialAttachments.find((attachment) => attachment.id === INDY_CREDENTIAL_ATTACHMENT_ID)
 
-    // Return null if attachment is not found
-    if (!attachment?.data?.base64) {
-      return null
-    }
-
     // Extract credential from attachment
-    const credentialJson = JsonEncoder.fromBase64(attachment.data.base64)
+    const credentialJson = attachment?.data ? attachment.data.getDataAsJson() : null
```

Comment:
Wouldn't this work? (also in the other places)

```suggestion
const credentialJson = attachment?.data?.getDataAsJson()
```

Reply:
I'll check that